Mathlib/CategoryTheory/Limits/HasLimits.lean

```diff
@@ -1047,125 +1038,112 @@ variable (F) [HasColimit F] (G : C ⥤ D) [HasColimit (F ⋙ G)]
   erw [colimit.ι_pre (F ⋙ G) E j, colimit.ι_post]
 #align category_theory.limits.colimit.pre_post CategoryTheory.Limits.colimit.pre_post
 
 open CategoryTheory.Equivalence
 
-instance hasColimitEquivalenceComp (e : K ≌ J) [HasColimit F] : HasColimit (e.Functor ⋙ F) :=
+instance hasColimit_equivalence_comp (e : K ≌ J) [HasColimit F] : HasColimit (e.functor ⋙ F) :=
   HasColimit.mk
-    { Cocone := Cocone.whisker e.Functor (Colimit.cocone F)
-      IsColimit := IsColimit.whiskerEquivalence (colimit.isColimit F) e }
-#align category_theory.limits.has_colimit_equivalence_comp CategoryTheory.Limits.hasColimitEquivalenceComp
+    { cocone := Cocone.whisker e.functor (colimit.cocone F)
+      isColimit := IsColimit.whiskerEquivalence (colimit.isColimit F) e }
+#align category_theory.limits.has_colimit_equivalence_comp CategoryTheory.Limits.hasColimit_equivalence_comp
 
 /-- If a `E ⋙ F` has a colimit, and `E` is an equivalence, we can construct a colimit of `F`.
 -/
-theorem hasColimitOfEquivalenceComp (e : K ≌ J) [HasColimit (e.Functor ⋙ F)] : HasColimit F :=
-  by
-  haveI : has_colimit (e.inverse ⋙ e.functor ⋙ F) := limits.has_colimit_equivalence_comp e.symm
-  apply has_colimit_of_iso (e.inv_fun_id_assoc F).symm
-#align category_theory.limits.has_colimit_of_equivalence_comp CategoryTheory.Limits.hasColimitOfEquivalenceComp
+theorem hasColimit_of_equivalence_comp (e : K ≌ J) [HasColimit (e.functor ⋙ F)] : HasColimit F := by
+  haveI : HasColimit (e.inverse ⋙ e.functor ⋙ F) := Limits.hasColimit_equivalence_comp e.symm
+  apply hasColimitOfIso (e.invFunIdAssoc F).symm
+#align category_theory.limits.has_colimit_of_equivalence_comp CategoryTheory.Limits.hasColimit_of_equivalence_comp
 
 section ColimFunctor
 
 variable [HasColimitsOfShape J C]
 
 section
 
-attribute [local simp] colim_map
+-- attribute [local simp] colimMap -- Porting note
 
 /-- `colimit F` is functorial in `F`, when `C` has all colimits of shape `J`. -/
-@[simps obj]
+@[simps] -- Porting note: simps on all fields now
 def colim : (J ⥤ C) ⥤ C where
   obj F := colimit F
-  map F G α := colimMap α
-  map_id' F := by
-    ext
-    erw [ι_colim_map, id_comp, comp_id]
-  map_comp' F G H α β := by
-    ext
-    erw [← assoc, is_colimit.fac, is_colimit.fac, assoc, is_colimit.fac, ← assoc]
-    rfl
+  map α := colimMap α
+  map_id F := by aesop_cat
+  map_comp α β := by aesop_cat
 #align category_theory.limits.colim CategoryTheory.Limits.colim
 
 end
 
-variable {F} {G : J ⥤ C} (α : F ⟶ G)
+variable {G : J ⥤ C} (α : F ⟶ G)
 
-@[simp, reassoc.1]
-theorem colimit.ι_map (j : J) : colimit.ι F j ≫ colim.map α = α.app j ≫ colimit.ι G j := by
-  apply is_colimit.fac
+-- @[reassoc (attr := simp)] Porting note: now simp can prove these
+theorem colimit.ι_map (j : J) : colimit.ι F j ≫ colim.map α = α.app j ≫ colimit.ι G j := by simp
 #align category_theory.limits.colimit.ι_map CategoryTheory.Limits.colimit.ι_map
 
-@[simp]
+@[simp] -- Porting note: adjusted this with full simps on colim
```

Comment:
What does this note mean?

Reply:
Per the discussion above with @adamtopaz, we are trying to move from `@[simps obj]` to `@[simps]` on `colim`. This proof had to be adjusted. Clarified the language